client/webserver/site/src/js/markets.ts

```diff
@@ -2904,3 +2996,38 @@ function hostColor (host: string): string {
   hosts.sort()
   return generateHue(hosts.indexOf(host))
 }
+
+// highlight will change element background color to red and back in a smooth
+// transition.
+function highlight (element: PageElement, color: string) {
+  if (element.style.backgroundColor === color) {
+    // If we are here, the element is already undergoing highlight transition,
+    // we must not interfere with it - if we do, it might break.
+    return
+  }
+
+  const previousBColor = element.style.backgroundColor
+  const prevTransition = element.style.transition
+
+  element.style.backgroundColor = color
+  element.style.transition = 'background 500ms'
```

Comment:
Another thing to consider is that the `transition` attribute applies both direction, which means we can only fade in -> fade out, we can't say flash on -> fade out, which I think would be a better indicator. 

A couple of suggestions:
1) Use `Doc.animate` to do a flash -> fade
2) For inputs, at least, animate the element's `outline` attribute rather than the background color.

Reply:
> why not specify it in CSS?

My knowledge of CSS and some front-end stuff is a bit rusty, I've rewrote this piece with `Doc.animate`; please let me know if you had a different solution in mind.

> The prevTransition stuff is likely unnecessary. Did you have a conflict with another transition value?

Not really, the example I was following was doing that - my guess would be it's just "general purpose code", sort of best practice; removed.

I've applied those suggestions, seem - looks better indeed!